global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl

```diff
@@ -98,13 +98,13 @@ route:
     match_re:
       tier: os
       severity: info|warning|critical
-      service: arc|backup|barbican|castellum|cinder|cfm|designate|elektra|elk|glance|hermes|ironic|keppel|keystone|limes|lyra|maia|manila|neutron|nova|octavia|sentry|swift
+      service: arc|backup|barbican|castellum|cinder|cfm|designate|elektra|elk|glance|hermes|ironic|keppel|keystone|limes|lyra|maia|manila|neutron|nova($|-.*)|octavia|sentry|swift
```

Comment:
Why is that needed?

Reply:
So SRE alerts für `nova-api` and `nova-console` and `nova-placement-api` get routed to `#cc-os-nova`, too.

Reply:
I see. Please be explicit in the config making this `nova|nova-api|nova-console|nova-placement-api`.

Reply:
Why? Everything starting with nova- should go to cc-os-nova. Why do we have to search for all possible services and always update this rule if we add one?

Reply:
Because we started with regexes a while ago which proved to become messy over time. Thus I want to keep the configuration explicit.

Reply:
Then I vote for removing this and trying to relabel the "service" to nova in the alerts.

Reply:
Or how about setting the `($|-.*)` at the end, so that it's valid for all services. Would that work for you? So that `arc-api` ingress also gets into the `arc` channel etc.

Reply:
Would also work. You could still use e.g. the `context` label for logical grouping of alerts.

Reply:
Please no regex-ing here anymore. I've spent too many hours chasing alerts.

Reply:
ok then, removal it is. @geisslet could you remove that, please? We will have to put the regexes into the Alert-Definition instead.